core/src/internal/temporal-util.ts

```diff
@@ -575,3 +580,12 @@ function formatNumber (
   }
   return isNegative ? '-' + numString : numString
 }
+
+function add (x: NumberOrInteger, y: number): NumberOrInteger {
+  if (x instanceof Integer) {
```

Comment:
any reason functions have a space before listing arguments in this file?

Reply:
It is in the repository codestyle and it's done automatically. But it's a bit useful when you're searching for methods use or declarations.